src/com/pentalog/pentastagiu/homework3/ex2/Basket.java

```diff
@@ -0,0 +1,51 @@
+package com.pentalog.pentastagiu.homework3.ex2;
+
+public class Basket {
+    private double totalPrice;
+    private Costumer costumer;
+
+    public Basket(Costumer costumer) {
+        this.costumer = costumer;
```

Comment:
si aici e nevoie de o validare si daca customer e null sa nu permiti instantierea(asta se intampla daca arunci o exceptie din constructor): astfel impiedici crearea de obiecte invalide si te asiguri ca nu ai probleme neasteptate mai tarziu(de ex in infoBasket, daca customer e null iti va fi aruncat un NullPointerException)